x-pack/test/functional/apps/infra/logs_source_configuration.ts

```diff
@@ -107,7 +108,9 @@ export default ({ getPageObjects, getService }: KibanaFunctionalTestDefaultProvi
       it('renders the changed log columns with their headers', async () => {
         const columnHeaderLabels = await infraLogStream.getColumnHeaderLabels();
 
-        expect(columnHeaderLabels).to.eql(['host.name', 'Timestamp', '']);
+        // TODO: make test more robust
+        // expect(columnHeaderLabels).to.eql(['host.name', 'Timestamp', '']);
+        expect(columnHeaderLabels).to.eql(['Timestamp', 'host.name', '']);
```

Comment:
Do we want this assertion?

Reply:
That assertion is for the test to see if log columns can successfully be added. It's not flaky since it's not a drag-and-drop/keyboard interaction like the reordering is.